cf-custom-resources/lib/wkld-cert-validator.js

```diff
@@ -0,0 +1,805 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require("aws-sdk");
+const CRYPTO = require("crypto");
+const ATTEMPTS_VALIDATION_OPTIONS_READY = 10;
+const ATTEMPTS_RECORD_SETS_CHANGE = 10;
+const DELAY_RECORD_SETS_CHANGE_IN_S = 30;
+const ATTEMPTS_CERTIFICATE_VALIDATED = 19;
+const ATTEMPTS_CERTIFICATE_NOT_IN_USE = 12;
+const DELAY_CERTIFICATE_VALIDATED_IN_S = 30;
+
+let envHostedZoneID, appName, envName, serviceName, certificateDomain, domainTypes, rootDNSRole, domainName, cloudFrontCert;
```

Comment:
nit: cuz seems like `cloudFrontCert` is a bool
```suggestion
let envHostedZoneID, appName, envName, serviceName, certificateDomain, domainTypes, rootDNSRole, domainName, isCloudFront;
```